graphql/schema.graphql

```diff
@@ -797,15 +801,185 @@ type GroupedProjects {
   projects: [Project!]!
 }
 
+type ProjectSettings {
+  githubWebhooksEnabled: Boolean!
+  projectRef: Project
+  vars: ProjectVars
+  aliases: [ProjectAlias]
+  subscriptions: [ProjectSubscription]
+}
+
+type ProjectVars {
+  vars: StringMap
+  privateVars: [String]
+}
+
+type ProjectAlias {
+  id: String!
+  alias: String!
+  gitTag: String!
+  variant: String!
+  task: String!
+  remotePath: String!
+  variantTags: [String!]!
+  taskTags: [String!]!
+}
+
+type ProjectSubscription {
+  id: String!
+  resourceType: String!
+  trigger: String!
+  selectors: [Selector]!
+  regexSelectors: [Selector]!
```

Comment:
Lets make Selector non nullable as well if the array is non nullable
`[Selector!]!`